docs/resource-editing.md

```diff
@@ -29,6 +29,14 @@ Resource links are marked in the gutter and underlined in the Editor, with corre
 Broken links are shown with yellow triangles in the Editor as in the Navigator:
 
 ![Source Editor Broken Links](img/source-editor-broken-links.png)
+
+Hover over the broken link in the Editor to show the options for working with the broken link:
+
+![Source Editor Broken Links](img/create-resource-from-unsatisfied-link-1.5.0.png)
+
```

Comment:
perhaps add something in the line of "Clicking the Create ... option in this popup will open the New Resource dialog pre-filled accordingly" ?

Reply:
Added.